products/ome-files/downloads/index.html

```diff
@@ -10,6 +10,182 @@
+                        <img alt="C++ logo" src="{{ site.baseurl }}/img/logos/3rd-party_cpp.png">
+                        <h5>OME Files</h5>
+                        <a href="http://github.com/ome/ome-files-cpp" class="button hollow tiny"><i class="fa fa-github"></i> View Repo</a>
+                        <a href="http://downloads.openmicroscopy.org/ome-files-cpp/0.3.2/source/ome-files-cpp-0.3.2.zip" class="button hollow tiny"><i class="fa fa-download"></i> Download Zip</a>
+                    </div>
+                </div>
+            </div>
+            <div class="small-12 medium-2 columns">                
+                <div class="card card-download-small">
+                    <div class="card-section">
+                        <img alt="C++ logo" src="{{ site.baseurl }}/img/logos/3rd-party_cpp.png">
+                        <h5 style="font-size: .99rem;">OME QtWidgets</h5>
+                        <a href="http://github.com/ome/ome-qtwidgets" class="button hollow tiny"><i class="fa fa-github"></i> View Repo</a>
+                        <a href="http://downloads.openmicroscopy.org/ome-qtwidgets/5.4.0/source/ome-qtwidgets-5.4.0.zip" class="button hollow tiny"><i class="fa fa-download"></i> Download Zip</a>
+                    </div>
+                </div>
+            </div>
+            <div class="small-12 medium-2 columns">                
+                <div class="card card-download-small">
+                    <div class="card-section">
+                        <img alt="OME logo" style="max-height: 90px;" src="{{ site.baseurl }}/img/logos/ome-logomark.svg">
+                        <h5 style="font-size: .96rem; margin: 10px 0 15px;">OME Super-Build</h5>
+                        <a href="http://github.com/ome/ome-cmake-superbuild" class="button hollow tiny"><i class="fa fa-github"></i> View Repo</a>
+                        <a href="http://downloads.openmicroscopy.org/ome-cmake-superbuild/0.3.2/source/ome-cmake-superbuild-0.3.2.zip" class="button hollow tiny"><i class="fa fa-download"></i> Download Zip</a>
+                    </div>
+                </div>
+            </div>
+        </div>
         <!-- end Download options -->
-        <hr class="whitespace">
\ No newline at end of file
+        <hr class="invisible">
+        
+        {% include back-to-top.html %}
+        
+        <!-- begin CTA -->
+        <hr class="whitespace">
+        <header class="marketing-hero" style="background-image: url('{{ site.baseurl }}/img/bg/ome-files.jpg');">
+            <div class="row">
+                <div class="small-12 columns bg-image-text-container-cta text-center">
+                    <p class="hero-subheader">All components available for this version of OME Files are available here:</p>
+                    <a href="http://downloads.openmicroscopy.org/ome-files-cpp/" class="large btn-blue button sites-button hide-for-small-only" style="text-shadow: none;">Download Components</a>
```

Comment:
Should be https://downloads.openmicroscopy.org/ome-files-cpp/0.3.2/22/xxx where xxx is `sources` and/or `tools`. In the case of multiple subdirectories, does it make sense to have a few of these buttons?